docs/images/esm-size.svg

```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="106" height="20"><linearGradient id="b" x2="0" y2="100%"><stop offset="0" stop-color="#bbb" stop-opacity=".1"/><stop offset="1" stop-opacity=".1"/></linearGradient><mask id="a"><rect width="106" height="20" rx="3" fill="#fff"/></mask><g mask="url(#a)"><path fill="#555" d="M0 0h57v20H0z"/><path fill="#e05d44" d="M57 0h49v20H57z"/><path fill="url(#b)" d="M0 0h106v20H0z"/></g><g fill="#fff" text-anchor="middle" font-family="DejaVu Sans,Verdana,Geneva,sans-serif" font-size="11"><text x="28.5" y="15" fill="#010101" fill-opacity=".3">esm-size</text><text x="28.5" y="14">esm-size</text><text x="80.5" y="15" fill="#010101" fill-opacity=".3">3.72 kB</text><text x="80.5" y="14">3.72 kB</text></g></svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" width="106" height="20"><linearGradient id="b" x2="0" y2="100%"><stop offset="0" stop-color="#bbb" stop-opacity=".1"/><stop offset="1" stop-opacity=".1"/></linearGradient><mask id="a"><rect width="106" height="20" rx="3" fill="#fff"/></mask><g mask="url(#a)"><path fill="#555" d="M0 0h57v20H0z"/><path fill="#e05d44" d="M57 0h49v20H57z"/><path fill="url(#b)" d="M0 0h106v20H0z"/></g><g fill="#fff" text-anchor="middle" font-family="DejaVu Sans,Verdana,Geneva,sans-serif" font-size="11"><text x="28.5" y="15" fill="#010101" fill-opacity=".3">esm-size</text><text x="28.5" y="14">esm-size</text><text x="80.5" y="15" fill="#010101" fill-opacity=".3">4.05 kB</text><text x="80.5" y="14">4.05 kB</text></g></svg>
```

Comment:
any reason no new line?

Reply:
This asset is actually created using a build tool... In the future we should swap it out for whatever Tram-One is using (which presents easier to read assets)